torchgeo/datamodules/deepglobelandcover.py

```diff
@@ -23,26 +29,45 @@ class DeepGlobeLandCoverDataModule(pl.LightningDataModule):
 
     def __init__(
         self,
-        batch_size: int = 64,
+        train_batch_size: int = 32,
         num_workers: int = 0,
+        patch_size: Union[Tuple[int, int], int] = (64, 64),
+        num_patches_per_tile: int = 32,
         val_split_pct: float = 0.2,
         **kwargs: Any,
     ) -> None:
-        """Initialize a LightningDataModule for DeepGlobe Land Cover based DataLoaders.
+        """Initialize a LightningDataModule for DeepGlobeLandCover based DataLoaders.
 
         Args:
-            batch_size: The batch size to use in all created DataLoaders
+            train_batch_size: The batch size used in the train DataLoader
+                (val_batch_size == test_batch_size == 1). The effective batch size
+                will be 'train_batch_size' * 'num_patches_per_tile'
             num_workers: The number of workers to use in all created DataLoaders
             val_split_pct: What percentage of the dataset to use as a validation set
+            patch_size: Size of random patch from image and mask (height, width), should
```

Comment:
"a multiple of 32" is a much better way of stating that "the numbers should be able to be evenly divisible by 2 5 times in a row" (i.e. what I usually say)